internal/cmd/operator-sdk/cli/cli.go

```diff
@@ -78,32 +70,6 @@ func Run() error {
 // This CLI can run kubebuilder commands and certain SDK specific commands that are aligned for
 // the kubebuilder project layout
 func GetPluginsCLIAndRoot() (*cli.CLI, *cobra.Command) {
-	// deprecated
```

Comment:
I see the "deprecated" notices attached to gov[23]Bundle. Should the strategy instead be "leave the deprecated bundles in place, but mark them deprecated" instead of "remove these"? I would take deprecation markings as an explicit vote to "keep these in place, so long as it's a low overhead maneuver to do so".

Reply:
With the `v4` version of `kubebuilder` there is no longer a `v3` or `v2` golang plugin, so we really can't lease these in place.

https://github.com/kubernetes-sigs/kubebuilder/tree/master/pkg/plugins/golang

Reply:
Got it. Feel free to resolve.